src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -311,6 +311,15 @@ void RegisterAppInterfaceRequest::Run() {
   SendRegisterAppInterfaceResponseToMobile();
 }
 
+void FillVRRelatedFields(smart_objects::SmartObject& response_params,
```

Comment:
@anosach-luxoft I used this function to reduce complexity of SendRegisterAppInterfaceResponseToMobile method. I think that we should have few such functions for UI related fields, Navi related fields, etc ... 
I should not be in anonymous name space because it should belong to am::request  namespace